aws/kops-aws-platform/iam-authenticator.tf

```diff
@@ -0,0 +1,55 @@
+variable "cluster_id" {
+  type        = "string"
+  description = "A unique-per-cluster identifier to prevent replay attacks. Good choices are a random token or a domain name that will be unique to your cluster"
+}
+
+variable "kube_config_path" {
+  type        = "string"
+  description = "Path to the kube config file. Can be sourced from `KUBE_CONFIG` or `KUBECONFIG`"
```

Comment:
So, if it can be sourced from `KUBE_CONFIG` or `KUBECONFIG`, then we should be able to default this to `""`